nuxt.config.ts

```diff
@@ -5,153 +5,160 @@ const SITE_DESCRIPTION
-                    name: 'twitter:image:alt',
-                    content: SITE_TITLE
-                },
-                {
-                    hid: 'og:title',
-                    property: 'og:title',
-                    content: SITE_TITLE
-                },
-                {
-                    hid: 'og:description',
-                    property: 'og:description',
-                    content: SITE_DESCRIPTION
-                },
-                {
-                    hid: 'og:image',
-                    property: 'og:image',
-                    content: 'https://www.findadoc.jp/findadoc-social.png'
-                },
-                {
-                    hid: 'og:image:secure_url',
-                    property: 'og:image:secure_url',
-                    content: 'https://www.findadoc.jp/findadoc-social.png'
-                },
-                {
-                    hid: 'og:image:alt',
-                    property: 'og:image:alt',
-                    content: SITE_TITLE
-                },
-                {
-                    hid: 'og:url',
-                    property: 'og:url',
-                    content: 'https://www.findadoc.jp'
-                }
-            ],
-            link: [{ rel: 'icon', type: 'image/x-icon', href: '/assets/favicon.ico' }]
-        }
-    },
-
-    // Global CSS: https://go.nuxtjs.dev/config-css
-    css: ['~/assets/css/tailwind.css'],
-
-    // Plugins to run before rendering page: https://go.nuxtjs.dev/config-plugins
-    plugins: [],
-
-    // Auto import components: https://go.nuxtjs.dev/config-components
-    components: true,
-
-    modules: [
-        '@nuxtjs/i18n',
-        '@pinia/nuxt',
-        'nuxt-viewport',
-        'nuxt-svgo',
-        '@nuxt/eslint',
-        '@nuxt/test-utils/module'
-    ],
-    eslint: {
-        config: {
-            stylistic: true
-        }
-    },
-    viewport: {
-        breakpoints: {
-            desktop: 1024,
-            desktopMedium: 1280,
-            desktopWide: 1600,
-
-            mobile: 320,
-            mobileMedium: 375,
-            mobileWide: 425,
-
-            tablet: 768
-        },
-
-        cookie: {
-            name: 'viewport'
-        },
-
-        defaultBreakpoints: {
-            desktop: 'desktop',
-            mobile: 'mobile',
-            tablet: 'tablet'
-        },
-
-        fallbackBreakpoint: 'desktop'
-    },
-
-    i18n: {
-        vueI18n: './i18n.options.ts'
-    },
-    svgo: {
-        defaultImport: 'component'
-    },
-    // Postcss configuration
-    postcss: {
-        plugins: {
-            tailwindcss: {},
-            autoprefixer: {}
-        }
-    },
-    runtimeConfig: {
-        public: {
-
-            GOOGLE_MAPS_API_KEY: process.env.GOOGLE_MAPS_API_KEY,
-
-            ENABLE_MODERATION_PANEL: process.env.ENABLE_MODERATION_PANEL,
-
-            NUXT_USE_LOCAL_API: process.env.NUXT_USE_LOCAL_API
-        }
-    },
-    telemetry: false
-})
+  app: {
```

Comment:
It's hard to tell what has actually been changed in this file, because the indentation has been changed, which makes it appear as if the whole file has been changed.